src/App.js

```diff
@@ -0,0 +1,35 @@
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+
+import Navbar from './components/Navbar';
+
+import Login from './pages/Login';
+import Registration from './pages/Registration';
+import Main from './pages/Main';
+import Vehicle from './pages/Vehicle';
+import MyReservations from './pages/MyReservations';
+import Reservation from './pages/Reservation';
+import Delete from './pages/Delete';
+import Add from './pages/Add';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main App functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will  meet the latest standards. 💪

Reply:
Thanks for the feedback, changes were implemented.